### PR TITLE
Add 2d hydrostatic column benchmark test

### PR DESCRIPTION
**Pull request by ezrayst**
Check gravity loading of a column, currently using newtonian
Provided results in `mpm-doc`

### CODE REVIEW

File: 2d/hydrostatic_column/mpm_usf_1.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "title" : "Column Collapse (Hydrostatic)",
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_usf_1.json]:
```suggestion
  "title" : "Hydrostatic column",
```

File: 2d/hydrostatic_column/mpm_usf_1.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "type" : "Newtonian2D",
+    "density" : 1800,
+    "bulk_modulus" : 1.0E+6,
+    "mu" : 0.0
+    }
+  ],
+  "analysis" : {
+    "type" : "MPMExplicitUSF2D",    
+    "dt" : 0.00001,
+    "uuid" : "hydrostatic_usf_1",
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_usf_1.json]:
```suggestion
    "uuid" : "hydrostatic_usf",
```

File: 2d/hydrostatic_column/mpm_usf_1.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "nsteps" : 10001,
+    "gravity": [0.0, -9.81],
+    "velocity_update" : true,
+    "resume" : {
+      "resume": false,
+      "uuid" : "hydrostatic_usf_1",
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_usf_1.json]:
```suggestion
      "uuid" : "hydrostatic_usf",
```

File: 2d/hydrostatic_column/mpm_usf_1.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_usf_1.json]: Rename to `mpm_usf.json` no need to have multiple files for velocity update and not. Just one of these is sufficient.

File: 2d/hydrostatic_column/mpm_usf_1.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    "type" : "MPMExplicitUSF2D",    
+    "dt" : 0.00001,
+    "uuid" : "hydrostatic_usf_1",
+    "nsteps" : 10001,
+    "gravity": [0.0, -9.81],
+    "velocity_update" : true,
+    "resume" : {
+      "resume": false,
+      "uuid" : "hydrostatic_usf_1",
+      "step" : 8000
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_usf_1.json]: Why step 8000?

File: 2d/hydrostatic_column/mpm_usf_1.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    "density" : 1800,
+    "bulk_modulus" : 1.0E+6,
+    "mu" : 0.0
+    }
+  ],
+  "analysis" : {
+    "type" : "MPMExplicitUSF2D",    
+    "dt" : 0.00001,
+    "uuid" : "hydrostatic_usf_1",
+    "nsteps" : 10001,
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_usf_1.json]: any particular reason for `nsteps` being 10001?

**Review by kks32 — CHANGES_REQUESTED** [review]
Keep just `mpm_usf_1.json` and rename to `mpm.json`

File: 2d/hydrostatic_column/mpm_usf_1.json (hunk 5 above)
Comment (ezrayst) [reply 2d/hydrostatic_column/mpm_usf_1.json]: I'm sorry, it was used for something else. `resume` is `false` anyway

**Comment by ezrayst** [comment]
I still keep `usf` and `usl` separately. Would this be OK, or do we simply want one file? The rest of the benchmarks have separated both though

**Comment by kks32** [comment]
One file with USF will be fine. No need for separate USF and USL

**Comment by ezrayst** [comment]
If the PR in `mpm-doc` is accepted, this can be accepted as well I think. The benchmark test files given are for `Case 1` (the coarse mesh)

**Comment by kks32** [comment]
@ezrayst Please make the requested changes

File: 2d/hydrostatic_column/mpm_linearelastic.json [hunk 7]
```diff
@@ -0,0 +1,41 @@
+{
+  "title" : "Hydrostatic Column",
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_linearelastic.json]:
```suggestion
  "title" : "Hydrostatic column",
```

File: 2d/hydrostatic_column/mpm_linearelastic.json [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+  "input_files" : {
+    "mesh" : "mesh.txt",
+    "velocity_constraints" : "velocity-constraints.txt",
+    "particles" : "particles.txt"
+  }, 
+  "mesh" : {
+    "cell_type": "ED2Q4",
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_linearelastic.json]: Does it have to be isoparametric? can we set that to false

**Review by kks32 — CHANGES_REQUESTED** [review]

File: 2d/hydrostatic_column/mpm_linearelastic.json [hunk 9]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_linearelastic.json]: Rename file to `mpm.json`

File: 2d/hydrostatic_column/mpm_linearelastic.json [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+  "mesh" : {
+    "cell_type": "ED2Q4",
+    "material_id": 0,
+    "mesh_reader": "Ascii2D",
+    "node_type": "N2D",
+    "particle_type": "P2D"
+  },
+  "materials" : [
+    {
+    "id" : 0,
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_linearelastic.json]: Add both materials here

File: 2d/hydrostatic_column/mpm_linearelastic.json [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+    "type" : "LinearElastic2D",
+    "density" : 1800,
+    "youngs_modulus": 1.0E+06,
+    "poisson_ratio" : 0.0
+    }
+  ],
+  "analysis" : {
+    "type" : "MPMExplicitUSF2D",    
+    "dt" : 0.00001,
+    "uuid" : "hydrostatic_usf_linearelastic",
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_linearelastic.json]: `hydrostatic_usf`

File: 2d/hydrostatic_column/mpm_newtonian.json [hunk 12]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm_newtonian.json]: Remove this

**Review by kks32 — CHANGES_REQUESTED** [review]

**Review by kks32 — CHANGES_REQUESTED** [review]
Could you add other mesh and particles file as well?

File: 2d/hydrostatic_column/mpm_linearelastic.json (hunk 8 above)
Comment (ezrayst) [reply 2d/hydrostatic_column/mpm_linearelastic.json]: Yes, will add this

File: 2d/hydrostatic_column/mpm.json [hunk 13]
```diff
@@ -0,0 +1,48 @@
+{
+  "title" : "Hydrostatic Column",
```
Comment (kks32) [inline 2d/hydrostatic_column/mpm.json]:
```suggestion
  "title" : "Hydrostatic column",
```

**Review by kks32 — APPROVED** [review]

**Pull request merged**